app/assets/javascripts/backbone/views/calendar_view.js.coffee

```diff
@@ -0,0 +1,21 @@
+class MS.Views.CalendarView extends Backbone.View
+
+  el: 'body'
+
+  initialize: (params) ->
+    # Initialize calendar
+    $('#calendar').fullCalendar({
+      eventRender: (event, element) ->
+        element.find('.fc-title').html(event.title)
+    })
+
+    @.renderCalEvents(params.events)
+
+  renderCalEvents: (events) ->
+    self = @;
+
+    $(events).each (key, calEvent) ->
```

Comment:
You can just use fat arrow here and you won't need self = @. You should never need that in CoffeeScript.